pubspec.yaml

```diff
@@ -9,7 +9,7 @@ environment:
   sdk: ">=2.17.0 <3.0.0"
 
 dependencies:
-  analyzer: '>=4.4.0 <6.0.0'
```

Comment:
I can't leave a comment on the right line - but we need to bump the version of `dart_style`

Reply:
Done